addons/website/views/snippets/s_unveil.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_unveil" name="Unveil">
+    <section class="s_unveil pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="11">
+                <div class="o_grid_item g-height-3 g-col-lg-12 col-lg-12" data-name="Box" style="z-index: 1; grid-area: 1 / 1 / 3 / 13;">
```

Comment:
You are using two `z-index: 1;` (the other is L12)

Also not sure about the name "Box", maybe we should keep the default "Column" or maybe use "Title" if really we want something custom